src/components/items/productId/BackButton.jsx

```diff
@@ -0,0 +1,13 @@
+import { Link } from 'react-router-dom';
+import styles from './BackButton.module.css';
+import { ReactComponent as Back } from 'assets/imgs/ic_back.svg';
+
+const BackButton = () => {
+  
+  return (
+    <div className={styles['container']}>
+      <Link to='/items'><button className={styles['back-button']}>목록으로 돌아가기<Back /></button></Link>
```

Comment:
P2:
link 태그안에 기능없는 button 태그를 사용하시기 보다 link태그를 원하시는 디자인으로 스타일링해서 사용하시길 추천드립니다.
의미없는 button, link태그는 가독성에 좋지 않습니다.

```suggestion
      <Link to='/items' className={styles['back-button']}>목록으로 돌아가기<Back /></Link>
```